repo_branch.go

```diff
@@ -81,6 +81,12 @@ func (repo *Repository) GetBranches() ([]string, error) {
 	return branches, nil
 }
 
+// DeleteBranch delete a branch by name on repository.
```

Comment:
Just curious, why we need `DeleteBranchOptions` struct? Can we just use a boolean so the method signature will be `DeleteBranch(name string, force bool)`